move_base/src/move_base.cpp

```diff
@@ -1304,5 +1323,18 @@ namespace move_base {
       behavior->newGoalReceived();
     }
   }
-
 };
+
+namespace srs {
+  void ControlLoopAnalyzer::compute(std::vector<float>& vec, float& maximum_miss)
+  {
+    if(vec.empty())
+    {
+      maximum_miss = 0.0;
+      return;
+    }
+
+    std::sort(vec.begin(), vec.end());
```

Comment:
Do we need to sort? Very few entries, so sorting is okay, but wouldn't it be simpler to use http://www.cplusplus.com/reference/algorithm/max_element/?